scapy/data.py

```diff
@@ -421,14 +446,15 @@ def reverse_lookup(self, name, case_sensitive=False):
         :returns: a dict of mac:tuples (Name, Extended Name)
         """
         if case_sensitive:
-            filtr = lambda x, l: any(x == z for z in l)
```

Comment:
Why did you change `==` to `in`?

Reply:
It's more convenient to match "Sun Microsystems" when just looking for "Sun".
This function already returned a dictionary of all MACs that match the OUI ordered by OUI, so it makes sense for the match to be wider.

Reply:
It makes. A separated PR will have make it easier to review =)